### PR TITLE
Block rate fix

### LINKED ISSUE #31: Rebase & Bond times are inaccurate
Due to the fact that the DAPP uses a constant for the block rate (see https://github.com/KlimaDAO/klimadao/blob/main/lib/constants/index.ts#L95) which is pinned at `2.5` seconds, and because this is not really how often the block rate actually works, the 'time until fully vested' value on the bond dapp can be fairly inaccurate.

I've seen this value be off by as much as 20 hours. The correct way to do this would be to poll polygonscan's API to check what the actual value should be, instead of hard-coding a constant.

### PR DESCRIPTION
**Pull request by Vienio99**
## Related Ticket

Resolves #31 

## Checklist

- [X] Building the site with `npm run build-site` works without errors
- [X] Building the app with `npm run build-app` works without errors
- [X] I formatted JS and TS files with running `npm run format-all`

### CODE REVIEW

**Comment by kaxline** [comment]
@Atmosfearful @Vienio99 Is there another use for Block Rate besides estimating time to rebase? If not, can't we just get the `endBlock` from the `epoch` and then query the `EstimateTimeInSec` of the `endBlock`?

**Comment by Atmosfearful** [comment]
> @Atmosfearful @Vienio99 Is there another use for Block Rate besides estimating time to rebase? If not, can't we just get the `endBlock` from the `epoch` and then query the `EstimateTimeInSec` of the `endBlock`?

Good point, we can use this to replace the constant values used in `ESTIMATED_DAILY_REBASES` and perhaps in the bonds as well. Me and @Vienio99 spoke about this on a call and he is gonna look into it.

Recap:
```
// number of blocks per rebase
EPOCH_INTERVAL = 11520;
rebaseRate = EPOCH_INTERVAL * blockRate;
rebasesPerDay = secondsPerDay / rebaseRate;
```

File: lib/utils/getEstimatedDailyRebases/index.ts [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import { EPOCH_INTERVAL } from "../../constants/index";
+
+// Calculates estimated daily rebases based on current block rate
+export async function getEstimatedDailyRebases(blockRate: number) {
```
Comment (Atmosfearful) [inline lib/utils/getEstimatedDailyRebases/index.ts:4]: this has no `await` so it doesn't need to be an `async` function. You can remove it out of the promises array here 
https://github.com/KlimaDAO/klimadao/pull/70/files#diff-b96f585d57b9419336b96f03d5d29b05d4b7d5855d5162ab0d73d2cca3d8afbdR59

And remove the `awaits` where used

**Review by Atmosfearful — CHANGES_REQUESTED** [review]
NICE, good call putting it in redux.
Just one quick change

**Comment by Atmosfearful** [comment]
If this has a significant impact on APY I will need to check with coms and policy team before merging

**Comment by Atmosfearful** [comment]
Closing this PR in favor of #90 I guess I didn't need to, oh well 🤷

**Pull request closed without merging**